[BUG]Pod keep restarting after Hscale

➜  ~ kbcli version
Kubernetes: v1.25.7-gke.1000
KubeBlocks: 0.5.0-beta.17
kbcli: 0.5.0-beta.17

```
➜  ~ kbcli cluster create yjtest1            --termination-policy=WipeOut            --node-labels '"dev=true"' --tolerations '"key=dev,value=true,operator=Equal,effect=NoSchedule","key=large,value=true,operator=Equal,effect=NoSchedule"'            --monitor=false --enable-all-logs=false --cluster-definition=apecloud-mysql --set cpu=100m,memory=0.5Gi,replicas=1,storage=1Gi  --namespace kubeblocks
Info: --cluster-version is not specified, ClusterVersion ac-mysql-8.0.30 is applied by default
Cluster yjtest1 created

➜  ~ kbcli cluster list -n kubeblocks
NAME      NAMESPACE    CLUSTER-DEFINITION   VERSION           TERMINATION-POLICY   STATUS    CREATED-TIME
yjtest1   kubeblocks   apecloud-mysql       ac-mysql-8.0.30   WipeOut              Running   May 05,2023 11:44 UTC+0800

➜  ~ kbcli cluster hscale yjtest1                 --components mysql                 --replicas 4 --namespace kubeblocks
Please type the name again(separate with white space when more than one): yjtest1
OpsRequest yjtest1-horizontalscaling-c9vkh created successfully, you can view the progress:
	kbcli cluster describe-ops yjtest1-horizontalscaling-c9vkh -n kubeblocks

➜  ~ k get pod -n kubeblocks
NAME                                                     READY   STATUS    RESTARTS      AGE
kb-addon-alertmanager-webhook-adaptor-69d9c45448-2vf49   2/2     Running   0             25h
kb-addon-grafana-987b677cb-rd8gg                         3/3     Running   0             25h
kb-addon-prometheus-alertmanager-0                       2/2     Running   0             25h
kb-addon-prometheus-server-0                             2/2     Running   0             25h
kb-addon-snapshot-controller-8649678b74-sn8w4            1/1     Running   1 (24h ago)   25h
kubeblocks-6b98bd7f68-7dznw                              1/1     Running   0             27m
yjtest1-mysql-0                                          4/4     Running   0             10m
yjtest1-mysql-1                                          4/4     Running   0             4m33s
yjtest1-mysql-2                                          4/4     Running   0             4m33s
yjtest1-mysql-3                                          4/4     Running   3 (56s ago)   4m33s

➜  ~ k logs yjtest1-mysql-3  -n kubeblocks
Defaulted container "mysql" out of: mysql, metrics, kb-checkrole, config-manager
leader=
followers=yjtest1-mysql-0
KB_POD_NAME=yjtest1-mysql-3
host=yjtest1-mysql-3.yjtest1-mysql-headless
no leader or self is leader, no need to call add.
cluster_info=yjtest1-mysql-0.yjtest1-mysql-headless:13306#9N;yjtest1-mysql-1.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-2.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-3.yjtest1-mysql-headless:13306#1N@4
KB_MYSQL_CLUSTER_UID=d2ddf57b-6a85-43d7-9333-09b9118c6fe2
docker-entrypoint.sh mysqld --defaults-file=/opt/mysql/my.cnf --cluster-start-index=1 --cluster-info="yjtest1-mysql-0.yjtest1-mysql-headless:13306#9N;yjtest1-mysql-1.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-2.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-3.yjtest1-mysql-headless:13306#1N@4" --cluster-id=1
2023-05-05 03:54:17+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server  started.
2023-05-05 03:54:22+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2023-05-05 03:54:22+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server  started.
'/var/lib/mysql/mysql.sock' -> '/var/run/mysqld/mysqld.sock'
2023-05-05 03:54:27+00:00 [Note] [Entrypoint]: Found restore file: /data/mysql/data//.restore
2023-05-05 03:54:27+00:00 [Note] [Entrypoint]: Reinitializing consensus meta: mysqld --defaults-file=/opt/mysql/my.cnf --cluster-start-index=1 --cluster-info=yjtest1-mysql-0.yjtest1-mysql-headless:13306#9N;yjtest1-mysql-1.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-2.yjtest1-mysql-headless:13306#1N;yjtest1-mysql-3.yjtest1-mysql-headless:13306#1N@4 --cluster-id=1 --cluster-id=1 --cluster-start-index=1 --cluster-recover-snapshot --cluster-force-change-meta=ON --upgrade-wesql
2023-05-05T03:54:30.042578Z 0 [Warning] [MY-011068] [Server] The syntax 'slave_exec_mode' is deprecated and will be removed in a future release. Please use replica_exec_mode instead.
2023-05-05T03:54:30.045364Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.30) starting as process 2452
2023-05-05T03:54:30.140634Z 0 [Warning] [MY-000054] [Server] World-writable config file '/data/mysql/data/auto.cnf' is ignored.
2023-05-05T03:54:30.140832Z 0 [Warning] [MY-010107] [Server] World-writable config file '/data/mysql/data/auto.cnf' has been removed.
2023-05-05T03:54:30.141295Z 0 [Warning] [MY-010075] [Server] No existing UUID has been found, so we assume that this is the first time that this server has been started. Generating a new UUID: 89b3eab6-eaf8-11ed-9531-b66cbed516ec.
2023-05-05T03:54:30.148508Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2023-05-05T03:54:33.344772Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
2023-05-05T03:54:35.652184Z 4 [System] [MY-013381] [Server] Server upgrade from '80030' to '80030' started.
2023-05-05T03:54:35.852116Z 4 [System] [MY-013381] [Server] Server upgrade from '80030' to '80030' completed.
2023-05-05T03:54:36.554211Z 0 [System] [MY-010229] [Server] Starting XA crash recovery...
2023-05-05T03:54:36.748009Z 0 [System] [MY-010232] [Server] XA crash recovery finished.
2023-05-05T03:54:36.839405Z 0 [Warning] [MY-000000] [Server] Recover consensus index is 19
2023-05-05T03:54:36.851670Z 0 [Warning] [MY-000000] [Server] Force change meta to system table successfully.
2023-05-05T03:54:37.843286Z 0 [System] [MY-010910] [Server] /usr/sbin/mysqld: Shutdown complete (mysqld 8.0.30)  WeSQL Server - GPL, Release 5, Revision 28f261a.
2023-05-05 03:54:38+00:00 [Note] [Entrypoint]: Consensus meta initialized
2023-05-05 03:54:38+00:00 [Note] [Entrypoint]: Recovery with temporary server
2023-05-05 03:54:38+00:00 [Note] [Entrypoint]: Starting temporary server with --cluster-recover-snapshot
2023-05-05 03:54:38+00:00 [Note] [Entrypoint]: Waiting for server startup
2023-05-05T03:54:41.844930Z 0 [Warning] [MY-011068] [Server] The syntax 'slave_exec_mode' is deprecated and will be removed in a future release. Please use replica_exec_mode instead.
2023-05-05T03:54:41.848197Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.30) starting as process 2560
2023-05-05T03:54:42.042077Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2023-05-05T03:54:44.344085Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
2023-05-05T03:54:46.852519Z 0 [System] [MY-010229] [Server] Starting XA crash recovery...
2023-05-05T03:54:47.045631Z 0 [System] [MY-010232] [Server] XA crash recovery finished.
2023-05-05T03:54:47.047353Z 0 [Warning] [MY-000000] [Server] Recover consensus index is 19
2023-05-05T03:54:47.245031Z 0 [Warning] [MY-000000] [Server] last log term is 2, last log term index is 19.
[2023-05-05 03:54:47.252377] [Default] EasyNet::onConnected server 1
[2023-05-05 03:54:47.254154] [Default] EasyNet::onConnected server 2
[2023-05-05 03:54:47.342134] [Default] EasyNet::onConnected server 3
[2023-05-05 03:54:49.070842] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:49.070842] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:54:49.070842] [Default] Server 4 : Epoch task currentEpoch(0) during requestVote
[2023-05-05 03:54:49.070842] [Default] Server 4 : Start new requestVote: new term(151)
[2023-05-05 03:54:49.078450] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(210) is bigger than me(151).
[2023-05-05 03:54:49.078450] [Default] Server 4 : new term(old:151,new:210) !!
[2023-05-05 03:54:49.078450] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:54:49.084346] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(1) from server 101 which has been deleted already!
[2023-05-05 03:54:49.078464] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(1) from server 101 which has been deleted already!
[2023-05-05 03:54:50.915496] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:50.915496] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:54:50.915496] [Default] Server 4 : Epoch task currentEpoch(1) during requestVote
[2023-05-05 03:54:50.915496] [Default] Server 4 : Start new requestVote: new term(211)
[2023-05-05 03:54:50.946790] [Default] Server 4 : server 1 refuse to let me became leader.
[2023-05-05 03:54:50.947058] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(2) from server 101 which has been deleted already!
[2023-05-05 03:54:50.947374] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(2) from server 101 which has been deleted already!
[2023-05-05 03:54:52.819891] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:52.819891] [Default] Server 4 : Paxos state change from CAND to CAND !!
[2023-05-05 03:54:52.819891] [Default] Server 4 : Epoch task currentEpoch(3) during requestVote
[2023-05-05 03:54:52.819891] [Default] Server 4 : Start new requestVote: new term(212)
[2023-05-05 03:54:52.827430] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(213) is bigger than me(212).
[2023-05-05 03:54:52.827430] [Default] Server 4 : new term(old:212,new:213) !!
[2023-05-05 03:54:52.827430] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:54:52.834284] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(3) from server 101 which has been deleted already!
[2023-05-05 03:54:52.827792] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(3) from server 101 which has been deleted already!
[2023-05-05 03:54:54.709862] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:54.709862] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:54:54.709862] [Default] Server 4 : Epoch task currentEpoch(4) during requestVote
[2023-05-05 03:54:54.709862] [Default] Server 4 : Start new requestVote: new term(214)
[2023-05-05 03:54:54.718231] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(215) is bigger than me(214).
[2023-05-05 03:54:54.718231] [Default] Server 4 : new term(old:214,new:215) !!
[2023-05-05 03:54:54.718231] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:54:54.724663] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(4) from server 101 which has been deleted already!
[2023-05-05 03:54:54.718441] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(4) from server 101 which has been deleted already!
[2023-05-05 03:54:56.614315] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:56.614315] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:54:56.614315] [Default] Server 4 : Epoch task currentEpoch(5) during requestVote
[2023-05-05 03:54:56.614315] [Default] Server 4 : Start new requestVote: new term(216)
[2023-05-05 03:54:56.621385] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(5) from server 101 which has been deleted already!
[2023-05-05 03:54:56.621408] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(5) from server 101 which has been deleted already!
[2023-05-05 03:54:56.623198] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(217) is bigger than me(216).
[2023-05-05 03:54:56.623198] [Default] Server 4 : new term(old:216,new:217) !!
[2023-05-05 03:54:56.623198] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:54:58.501919] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:54:58.501919] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:54:58.501919] [Default] Server 4 : Epoch task currentEpoch(6) during requestVote
[2023-05-05 03:54:58.501919] [Default] Server 4 : Start new requestVote: new term(218)
[2023-05-05 03:54:58.509493] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(6) from server 101 which has been deleted already!
[2023-05-05 03:54:58.509515] [Default] Server 4 : server 1 refuse to let me became leader.
[2023-05-05 03:54:58.509675] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(6) from server 101 which has been deleted already!
[2023-05-05 03:55:00.400037] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:00.400037] [Default] Server 4 : Paxos state change from CAND to CAND !!
[2023-05-05 03:55:00.400037] [Default] Server 4 : Epoch task currentEpoch(8) during requestVote
[2023-05-05 03:55:00.400037] [Default] Server 4 : Start new requestVote: new term(219)
[2023-05-05 03:55:00.407480] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(220) is bigger than me(219).
[2023-05-05 03:55:00.407480] [Default] Server 4 : new term(old:219,new:220) !!
[2023-05-05 03:55:00.407480] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:00.414012] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(7) from server 101 which has been deleted already!
[2023-05-05 03:55:00.407675] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(7) from server 101 which has been deleted already!
[2023-05-05 03:55:02.290288] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:02.290288] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:02.290288] [Default] Server 4 : Epoch task currentEpoch(9) during requestVote
[2023-05-05 03:55:02.290288] [Default] Server 4 : Start new requestVote: new term(221)
[2023-05-05 03:55:02.299329] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(222) is bigger than me(221).
[2023-05-05 03:55:02.299329] [Default] Server 4 : new term(old:221,new:222) !!
[2023-05-05 03:55:02.299329] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:02.306257] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(8) from server 101 which has been deleted already!
[2023-05-05 03:55:02.299839] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(8) from server 101 which has been deleted already!
[2023-05-05 03:55:04.239429] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:04.239429] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:04.239429] [Default] Server 4 : Epoch task currentEpoch(10) during requestVote
[2023-05-05 03:55:04.239429] [Default] Server 4 : Start new requestVote: new term(223)
[2023-05-05 03:55:04.247117] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(224) is bigger than me(223).
[2023-05-05 03:55:04.247117] [Default] Server 4 : new term(old:223,new:224) !!
[2023-05-05 03:55:04.247117] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:04.247545] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(9) from server 101 which has been deleted already!
[2023-05-05 03:55:04.253881] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(9) from server 101 which has been deleted already!
[2023-05-05 03:55:06.077836] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:06.077836] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:06.077836] [Default] Server 4 : Epoch task currentEpoch(11) during requestVote
[2023-05-05 03:55:06.077836] [Default] Server 4 : Start new requestVote: new term(225)
[2023-05-05 03:55:06.084569] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(226) is bigger than me(225).
[2023-05-05 03:55:06.084569] [Default] Server 4 : new term(old:225,new:226) !!
[2023-05-05 03:55:06.084569] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:06.084911] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(10) from server 101 which has been deleted already!
[2023-05-05 03:55:06.085021] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(10) from server 101 which has been deleted already!
[2023-05-05 03:55:07.910859] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:07.910859] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:07.910859] [Default] Server 4 : Epoch task currentEpoch(12) during requestVote
[2023-05-05 03:55:07.910859] [Default] Server 4 : Start new requestVote: new term(227)
[2023-05-05 03:55:07.918147] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(228) is bigger than me(227).
[2023-05-05 03:55:07.918147] [Default] Server 4 : new term(old:227,new:228) !!
[2023-05-05 03:55:07.918147] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:07.918154] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(11) from server 101 which has been deleted already!
[2023-05-05 03:55:07.918428] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(11) from server 101 which has been deleted already!
[2023-05-05 03:55:09.749520] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:09.749520] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:09.749520] [Default] Server 4 : Epoch task currentEpoch(13) during requestVote
[2023-05-05 03:55:09.749520] [Default] Server 4 : Start new requestVote: new term(229)
[2023-05-05 03:55:09.756896] [Default] Server 4 : server 1 refuse to let me became leader.
[2023-05-05 03:55:09.756966] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(12) from server 101 which has been deleted already!
[2023-05-05 03:55:09.757175] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(12) from server 101 which has been deleted already!
[2023-05-05 03:55:11.558913] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:11.558913] [Default] Server 4 : Paxos state change from CAND to CAND !!
[2023-05-05 03:55:11.558913] [Default] Server 4 : Epoch task currentEpoch(15) during requestVote
[2023-05-05 03:55:11.558913] [Default] Server 4 : Start new requestVote: new term(230)
[2023-05-05 03:55:11.568555] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(231) is bigger than me(230).
[2023-05-05 03:55:11.568555] [Default] Server 4 : new term(old:230,new:231) !!
[2023-05-05 03:55:11.568555] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:11.568783] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(13) from server 101 which has been deleted already!
[2023-05-05 03:55:11.568883] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(13) from server 101 which has been deleted already!
[2023-05-05 03:55:13.423033] [Default] Server 4 : Enter startElectionCallback
[2023-05-05 03:55:13.423033] [Default] Server 4 : Paxos state change from FOLL to CAND !!
[2023-05-05 03:55:13.423033] [Default] Server 4 : Epoch task currentEpoch(16) during requestVote
[2023-05-05 03:55:13.423033] [Default] Server 4 : Start new requestVote: new term(232)
[2023-05-05 03:55:13.433111] [Default] Server 4 : New Term in onRequestVoteResponce !! server 1 's term(233) is bigger than me(232).
[2023-05-05 03:55:13.433111] [Default] Server 4 : new term(old:232,new:233) !!
[2023-05-05 03:55:13.433111] [Default] Server 4 : Paxos state change from CAND to FOLL !!
[2023-05-05 03:55:13.433588] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(14) from server 101 which has been deleted already!
[2023-05-05 03:55:13.433626] [ERROR] Server 4 : onRequestVoteResponce receive a msg msgId(14) from server 101 which has been deleted already!
2023-05-05 03:55:14+00:00 [ERROR] [Entrypoint]: Unable to start server.


➜  ~ k describe pod yjtest1-mysql-3 -n kubeblocks
Name:         yjtest1-mysql-3
Namespace:    kubeblocks
Priority:     0
Node:         gke-yjtest-default-pool-ee024711-n5m4/10.128.0.20
Start Time:   Fri, 05 May 2023 11:50:33 +0800
Labels:       app.kubernetes.io/component=mysql
              app.kubernetes.io/instance=yjtest1
              app.kubernetes.io/managed-by=kubeblocks
              app.kubernetes.io/name=apecloud-mysql
              app.kubernetes.io/version=ac-mysql-8.0.30
              apps.kubeblocks.io/component-name=mysql
              apps.kubeblocks.io/workload-type=Consensus
              controller-revision-hash=yjtest1-mysql-68786f76c8
              statefulset.kubernetes.io/pod-name=yjtest1-mysql-3
Annotations:  cs.apps.kubeblocks.io/leader:
Status:       Running
IP:           10.104.2.130
IPs:
  IP:           10.104.2.130
Controlled By:  StatefulSet/yjtest1-mysql
Containers:
  mysql:
    Container ID:  containerd://dc6aeb5bfb3eb65f368362bc6c92492d38aba180afcfa22087d17f53d6f1edd9
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server:8.0.30-5.alpha5.20230319.g28f261a.5
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server@sha256:057de8f4cae1815fdbed80cfff98f7b9c9ef9aea7d1bbbc768e6fc7e1846532c
    Ports:         3306/TCP, 13306/TCP
    Host Ports:    0/TCP, 0/TCP
    Command:
      /scripts/setup.sh
    State:          Running
      Started:      Fri, 05 May 2023 11:55:56 +0800
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Fri, 05 May 2023 11:54:17 +0800
      Finished:     Fri, 05 May 2023 11:55:14 +0800
    Ready:          True
    Restart Count:  4
    Limits:
      cpu:     100m
      memory:  512Mi
    Requests:
      cpu:     100m
      memory:  512Mi
    Environment Variables from:
      yjtest1-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:            yjtest1-mysql-3 (v1:metadata.name)
      KB_NAMESPACE:           kubeblocks (v1:metadata.namespace)
      KB_SA_NAME:              (v1:spec.serviceAccountName)
      KB_NODENAME:             (v1:spec.nodeName)
      KB_HOST_IP:              (v1:status.hostIP)
      KB_POD_IP:               (v1:status.podIP)
      KB_POD_IPS:              (v1:status.podIPs)
      KB_HOSTIP:               (v1:status.hostIP)
      KB_PODIP:                (v1:status.podIP)
      KB_PODIPS:               (v1:status.podIPs)
      KB_CLUSTER_NAME:        yjtest1
      KB_COMP_NAME:           mysql
      KB_CLUSTER_COMP_NAME:   yjtest1-mysql
      KB_POD_FQDN:            $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      MYSQL_ROOT_HOST:        %
      MYSQL_ROOT_USER:        <set to the key 'username' in secret 'yjtest1-conn-credential'>  Optional: false
      MYSQL_ROOT_PASSWORD:    <set to the key 'password' in secret 'yjtest1-conn-credential'>  Optional: false
      MYSQL_DATABASE:         mydb
      MYSQL_USER:             u1
      MYSQL_PASSWORD:         u1
      CLUSTER_ID:             1
      CLUSTER_START_INDEX:    1
      REPLICATION_USER:       replicator
      REPLICATION_PASSWORD:
      MYSQL_TEMPLATE_CONFIG:
      MYSQL_CUSTOM_CONFIG:
      MYSQL_DYNAMIC_CONFIG:
    Mounts:
      /data/mysql from data (rw)
      /etc/annotations from annotations (rw)
      /opt/mysql from mysql-config (rw)
      /scripts from scripts (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-fm25w (ro)
  metrics:
    Container ID:  containerd://5e3919fab4bc0716f53a45987b1662d673d4fa00d442447ebbe1e29702150416
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter:0.14.1
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter@sha256:8a394d7e722a3c6be7082485eafb4c4082af6f2caf49b30446472ffeedc2ea28
    Port:          9104/TCP
    Host Port:     0/TCP
    Command:
      /bin/mysqld_exporter
      --collect.info_schema.wesql_consensus
    State:          Running
      Started:      Fri, 05 May 2023 11:50:41 +0800
    Ready:          True
    Restart Count:  0
    Liveness:       http-get http://:http-metrics/ delay=5s timeout=5s period=10s #success=1 #failure=6
    Readiness:      http-get http://:http-metrics/ delay=5s timeout=5s period=10s #success=1 #failure=6
    Environment Variables from:
      yjtest1-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:           yjtest1-mysql-3 (v1:metadata.name)
      KB_NAMESPACE:          kubeblocks (v1:metadata.namespace)
      KB_SA_NAME:             (v1:spec.serviceAccountName)
      KB_NODENAME:            (v1:spec.nodeName)
      KB_HOST_IP:             (v1:status.hostIP)
      KB_POD_IP:              (v1:status.podIP)
      KB_POD_IPS:             (v1:status.podIPs)
      KB_HOSTIP:              (v1:status.hostIP)
      KB_PODIP:               (v1:status.podIP)
      KB_PODIPS:              (v1:status.podIPs)
      KB_CLUSTER_NAME:       yjtest1
      KB_COMP_NAME:          mysql
      KB_CLUSTER_COMP_NAME:  yjtest1-mysql
      KB_POD_FQDN:           $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      MYSQL_USER:            <set to the key 'username' in secret 'yjtest1-conn-credential'>  Optional: false
      MYSQL_PASSWORD:        <set to the key 'password' in secret 'yjtest1-conn-credential'>  Optional: false
      DATA_SOURCE_NAME:      $(MYSQL_USER):$(MYSQL_PASSWORD)@(localhost:3306)/
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-fm25w (ro)
  kb-checkrole:
    Container ID:  containerd://5f3f2289d006e3a661c6292999418d8c2fe233d7253a2e03b4ff1fdfabd4090e
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.5.0-beta.17
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools@sha256:b18de11a527d945ba4dae9fcda8e479ba7a546d162d9ab54f041db78083af1d7
    Ports:         3501/TCP, 50001/TCP
    Host Ports:    0/TCP, 0/TCP
    Command:
      probe
      --app-id
      batch-sdk
      --dapr-http-port
      3501
      --dapr-grpc-port
      50001
      --app-protocol
      http
      --log-level
      info
      --config
      /config/probe/config.yaml
      --components-path
      /config/probe/components
    State:          Running
      Started:      Fri, 05 May 2023 11:50:41 +0800
    Ready:          True
    Restart Count:  0
    Readiness:      exec [curl -X POST --max-time 1 --fail-with-body --silent -H Content-ComponentDefRef: application/json http://localhost:3501/v1.0/bindings/mysql -d {"operation": "checkRole", "metadata":{"sql":""}}] delay=0s timeout=1s period=1s #success=1 #failure=2
    Startup:        tcp-socket :3501 delay=0s timeout=1s period=10s #success=1 #failure=3
    Environment Variables from:
      yjtest1-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:                yjtest1-mysql-3 (v1:metadata.name)
      KB_NAMESPACE:               kubeblocks (v1:metadata.namespace)
      KB_SA_NAME:                  (v1:spec.serviceAccountName)
      KB_NODENAME:                 (v1:spec.nodeName)
      KB_HOST_IP:                  (v1:status.hostIP)
      KB_POD_IP:                   (v1:status.podIP)
      KB_POD_IPS:                  (v1:status.podIPs)
      KB_HOSTIP:                   (v1:status.hostIP)
      KB_PODIP:                    (v1:status.podIP)
      KB_PODIPS:                   (v1:status.podIPs)
      KB_CLUSTER_NAME:            yjtest1
      KB_COMP_NAME:               mysql
      KB_CLUSTER_COMP_NAME:       yjtest1-mysql
      KB_POD_FQDN:                $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      KB_SERVICE_USER:            <set to the key 'username' in secret 'yjtest1-conn-credential'>  Optional: false
      KB_SERVICE_PASSWORD:        <set to the key 'password' in secret 'yjtest1-conn-credential'>  Optional: false
      KB_SERVICE_PORT:            3306
      KB_SERVICE_ROLES:           {"follower":"Readonly","leader":"ReadWrite","learner":"Readonly"}
      KB_SERVICE_CHARACTER_TYPE:  mysql
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-fm25w (ro)
  config-manager:
    Container ID:  containerd://9ef64a4b7655ca0e2c2460d1382a311317f5181ee109e52b2c420acd3ef32029
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.5.0-beta.17
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools@sha256:b18de11a527d945ba4dae9fcda8e479ba7a546d162d9ab54f041db78083af1d7
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/reloader
    Args:
      --operator-update-enable
      --log-level
      info
      --tcp
      9901
      --notify-type
      tpl
      --tpl-config
      /opt/config/reload/reload.yaml
    State:          Running
      Started:      Fri, 05 May 2023 11:50:41 +0800
    Ready:          True
    Restart Count:  0
    Environment:
      CONFIG_MANAGER_POD_IP:   (v1:status.podIP)
      DB_TYPE:                mysql
      MYSQL_USER:             <set to the key 'username' in secret 'yjtest1-conn-credential'>  Optional: false
      MYSQL_PASSWORD:         <set to the key 'password' in secret 'yjtest1-conn-credential'>  Optional: false
      DATA_SOURCE_NAME:       $(MYSQL_USER):$(MYSQL_PASSWORD)@(localhost:3306)/
    Mounts:
      /opt/config/reload from reload-manager-reload (rw)
      /opt/mysql from mysql-config (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-fm25w (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             True
  ContainersReady   True
  PodScheduled      True
Volumes:
  data:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  data-yjtest1-mysql-3
    ReadOnly:   false
  annotations:
    Type:  DownwardAPI (a volume populated by information about the pod)
    Items:
      metadata.annotations['cs.apps.kubeblocks.io/leader'] -> leader
  scripts:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      yjtest1-mysql-apecloud-mysql-scripts
    Optional:  false
  mysql-config:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      yjtest1-mysql-mysql-consensusset-config
    Optional:  false
  reload-manager-reload:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      mysql-reload-script-yjtest1
    Optional:  false
  kube-api-access-fm25w:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Burstable
Node-Selectors:              <none>
Tolerations:                 dev=true:NoSchedule
                             kb-data=true:NoSchedule
                             large=true:NoSchedule
                             node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason                  Age                   From                     Message
  ----     ------                  ----                  ----                     -------
  Normal   Scheduled               5m40s                 default-scheduler        Successfully assigned kubeblocks/yjtest1-mysql-3 to gke-yjtest-default-pool-ee024711-n5m4
  Normal   SuccessfulAttachVolume  5m35s                 attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-2becc0cd-803b-4c72-bf5c-89bb2c7144e5"
  Normal   Started                 5m32s                 kubelet                  Started container kb-checkrole
  Normal   Started                 5m32s                 kubelet                  Started container config-manager
  Normal   Created                 5m32s                 kubelet                  Created container config-manager
  Normal   Pulled                  5m32s                 kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.5.0-beta.17" already present on machine
  Normal   Pulled                  5m32s                 kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter:0.14.1" already present on machine
  Normal   Created                 5m32s                 kubelet                  Created container metrics
  Normal   Started                 5m32s                 kubelet                  Started container metrics
  Normal   Pulled                  5m32s                 kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.5.0-beta.17" already present on machine
  Normal   Created                 5m32s                 kubelet                  Created container kb-checkrole
  Normal   AddAnnotation           5m23s (x7 over 6m2s)  pod-controller           add annotation cs.apps.kubeblocks.io/leader=yjtest1-mysql-0
  Normal   Started                 116s (x4 over 5m32s)  kubelet                  Started container mysql
  Normal   Created                 116s (x4 over 5m32s)  kubelet                  Created container mysql
  Normal   Pulled                  116s (x4 over 5m32s)  kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server:8.0.30-5.alpha5.20230319.g28f261a.5" already present on machine
  Warning  BackOff                 30s (x6 over 3m37s)   kubelet                  Back-off restarting failed container
```
[pod2log.txt](https://github.com/apecloud/kubeblocks/files/11403190/pod2log.txt)
[pod1log.txt](https://github.com/apecloud/kubeblocks/files/11403191/pod1log.txt)
[pod0log.txt](https://github.com/apecloud/kubeblocks/files/11403192/pod0log.txt)
[pod3log.txt](https://github.com/apecloud/kubeblocks/files/11403189/pod3log.txt)

--- a/deploy/apecloud-mysql/values.yaml
+++ b/deploy/apecloud-mysql/values.yaml
@@ -7,7 +7,7 @@ image:
   repository: apecloud/apecloud-mysql-server
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 8.0.30-5.alpha5.20230319.g28f261a.5
+  tag: 8.0.30-5.alpha8.20230523.g3e93ae7.5
 
 ## MySQL Cluster parameters
 cluster:

